src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/CaseServiceImpl.java

```diff
@@ -268,54 +261,22 @@ private void recordRateLimiting(
   }
 
   /*
-   * Call the rate limiter within a circuit-breaker, thus protecting the RHSvc
-   * functionality from the unlikely event that that the rate limiter service is failing.
+   * Call the rate limiter. The RateLimiterClient invokes the EnvoyLimiter within a circuit-breaker,
+   * thus protecting the RHSvc in the unlikely event that that the rate limiter service is failing.
```

Comment:
just an old typo to fix here

```suggestion
   * thus protecting the RHSvc in the unlikely event that the rate limiter service is failing.
```